Dockerfile

```diff
@@ -1,14 +1,12 @@
-FROM golang:1.11
+FROM golang:1.11 as builder
+WORKDIR /go/src/github.com/kshitij10496/hercules/
+ADD . /go/src/github.com/kshitij10496/hercules
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o hercules .
 
+FROM alpine:latest  
+RUN apk --no-cache add ca-certificates
 ENV PORT=8080
 ENV HERCULES_DATABASE="user=kshitij10496 dbname=herculesdb sslmode=disable"
-
-RUN mkdir -p /go/src/github.com/kshitij10496/hercules
-WORKDIR /go/src/github.com/kshitij10496/hercules
-ADD . /go/src/github.com/kshitij10496/hercules
-
-RUN go build .
-
-CMD ["./hercules"]
-
-EXPOSE 8080
```

Comment:
I guess this should be included in the new docker file as well.

BTW, thanks for the PR... it's something really useful for the production.
It would be a lot wiser from next time to include how the change works. Here's the link for this time:
https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds

cc/ @kshitij10496

Reply:
I edited the description of the PR to explain this modification.
I also fixed the forgotten `EXPOSE` directive.

Thanks for your review